fourmolu.yaml

```diff
@@ -0,0 +1,11 @@
+indentation: 2
+comma-style: leading # for lists, tuples etc. - can also be 'leading'
+import-export-style: leading
```

Comment:
apparently there is a `diff-friendly` style which I don’t find more friendly than `leading`.

Reply:
It's friendly when modifying the beginning of the import/export list. Admittedly perhaps not a common use-case, but nice when it happens